test/rtos_drivers/wifi/check_wifi.sh

```diff
@@ -73,5 +73,3 @@ echo "****************"
 echo "* Parse Result *"
 echo "****************"
 python ${REPO_ROOT}/test/rtos_drivers/python/parse_test_output.py testing/test.rpt -outfile="testing/test_results" --print_test_results --verbose
-
-pytest ${REPO_ROOT}/test/rtos_drivers/wifi
```

Comment:
Why has this test been removed?

Reply:
This test was moved in https://github.com/xmos/fwk_rtos/pull/192, and now it is run from the [Jenkinsfile](https://github.com/xmos/fwk_rtos/blob/develop/Jenkinsfile#L117).

Reply:
Thanks for the explanation.